pyoozie/oozie_client.py

```diff
@@ -0,0 +1,571 @@
+# Copyright (c) 2017 "Shopify inc." All rights reserved.
```

Comment:
Could we move this module to `pyoozie/api/_client.py` to make it private and put it under `api` to reduce ambiguity?